test/CMakeLists.txt

```diff
@@ -815,95 +813,103 @@ set(IMPLICITGEMM_TESTING_ENV
 )
 
 if(WORKAROUND_ISSUE_936 AND MIOPEN_TEST_HALF)
+    SET(SAVE_IMPLICITGEMM_TESTING_ENV ${IMPLICITGEMM_TESTING_ENV})
     LIST(APPEND IMPLICITGEMM_TESTING_ENV MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_HIP_FWD_V4R1=0 MIOPEN_FIND_MODE=normal)
-    LIST(APPEND IMPLICITGEMM_ARGS --disable-forward --disable-backward-data)
+    SET(SAVE_MIOPEN_TEST_FLOAT_ARG ${MIOPEN_TEST_FLOAT_ARG})
+    LIST(APPEND MIOPEN_TEST_FLOAT_ARG --disable-forward --disable-backward-data)
```

Comment:
The potential issue is that we modify IMPLICITGEMM_TESTING_ENV and IMPLICITGEMM_ARGS when WORKAROUND_ISSUE_936 is enabled, but do not restore them after `test_conv_for_implicit_gemm`.